frontend/integration-tests/tests/storage.scenario.ts

```diff
@@ -54,6 +54,7 @@ describe('Add storage is applicable for all workloads', () => {
         expect($(`[data-id="${pvcName}-${mountPath}"] [data-test-id="path"]`).getText()).toContain(
           mountPath,
         );
+        execSync(`kubectl delete --cascade ${resourceType} example -n ${testName}`);
```

Comment:
@spadgett This appears to be a double delete, and the other case has a try..catch; thoughts?

https://github.com/openshift/console/pull/3847/files#diff-cbca0dc593db54f1952a611b25cf49d0R27
```
    resourceObjs.forEach((resourceType) => {
      try {
        execSync(`kubectl delete --cascade ${resourceType} example -n ${testName}`);
      } catch (error) {
        console.error(`Failed to delete ${resourceType} example:\n${error}`);
      }
    });
```

Reply:
I added that temporarily to debug something. Let me add WIP to the title